Misc/NEWS.d/next/Tests/2024-02-23-01-35-25.gh-issue-115836.GDw5Ut.rst

```diff
@@ -0,0 +1 @@
+Stop using hardcoded line numbers in ``test_monitoring``.
```

Comment:
We don't really need this for a non-user-facing change. This PR is a reasonable case for the skip-news label (which I know you can't currently apply yourself...)

Reply:
Thanks, I was wondering about when that's appropriate. I almost asked someone to add the label but it was quicker to add a news entry with blurb-it :). I'll keep this in mind for the future.